commitlint.config.ts

```diff
@@ -158,6 +158,18 @@ module.exports = {
                     ];
                 },
 
+                'empty-wip': ({header}: {header:any}) => {
+                    let headerStr = convertAnyToString(header, "header");
+                    let offence = false;
+                    if (headerStr.toLowerCase() === "wip") {
+                        offence = true;
+                    }
```

Comment:
@parhamsaremi these 4 lines can be simplified to 1 line, and without mutation

Reply:
right I'll fix it right now.